tests/sqllogictests/Cargo.toml

```diff
@@ -20,7 +20,7 @@ common-exception = { path = "../../src/common/exception" }
 env_logger = "0.10.0"
 futures-util = "0.3.25"
 lazy_static = { workspace = true }
-mysql_async = { version = "0.31", default-features = false, features = ["rustls-tls"] }
+mysql_async = { version = "0.31", default-features = false, features = ["native-tls-tls"] }
```

Comment:
It's better not to use native-tls.

Reply:
It's just for testing purposes, I think it's acceptable, or I can write a patch to remove webpki.

Reply:
> It's just for testing purposes, I think it's acceptable

Query Service also changed which means our binary will need openssl. I prefer to ignore this audit warning.

Reply:
> Query Service also changed which means our binary will need openssl. I prefer to ignore this audit warning.

That's too bad, I'll write a patch.

Reply:
Maybe we can fork webpki to https://github.com/tent-rs?

Reply:
> Maybe we can fork webpki to https://github.com/tent-rs?

The approach upstream is to use webpki-roots, as webpki has been completely abandoned. I just need to remove it from mysql_async.

Reply:
> The approach upstream is to use webpki-roots, as webpki has been completely abandoned. I just need to remove it from mysql_async.

Makes sense